Library/Homebrew/cmd/install.rb

```diff
@@ -84,6 +84,8 @@ def install_args
                           "package into a Homebrew package."
       switch "-g", "--git",
              description: "Create a Git repository, useful for creating patches to the software."
+      switch "--download-silently",
+             description: "Disable download progress output."
```

Comment:
@MikeMcQuaid Are you satisfied with the name of this switch and the description?

Reply:
When using the `update` command it calls `fi.install` which is a method on `FormulaInstaller`. Inside the `FormulaInstaller` the `build` method creates a new process and calls the `install` command passing in arguments in order to isolate the environment. The only way to pass arguments was to create a command line flag for the `install` command and pass that flag from the `build` method into the new process.

Do you have an alternative implementation that would work?

Reply:
> When using the `update` command

(`upgrade`?)
> Do you have an alternative implementation that would work?

I'll have a look/think 👍

Reply:
> (upgrade?)
yes, that's correct.